encrypt.js

```diff
@@ -0,0 +1,27 @@
+const hkdf = require('futoin-hkdf');
+const { JWK, JWE } = require('jose');
+
+const BYTE_LENGTH = 32;
+const ENCRYPTION_INFO = 'JWE CEK';
+const options = { hash: 'SHA-256' };
+
+const deriveKey = (secret) => hkdf(secret, BYTE_LENGTH, { info: ENCRYPTION_INFO, ...options });
+
+module.exports = {
+  encrypt(arg) {
```

Comment:
I think it's a little more future-proof if you export a function from this module and assign the function inside `on('task' {})`. For example:

```js
// encrypt.js
module.exports = function encrypt(arg) {
}
```

```js
// cypress/plugins/index.js
const encrypt = require('../../encrypt');

module.exports = (on, config) => {
  on('task', {
    encrypt,
  });
}
```

I say this because it will make it easier to add additional tasks to this library and it makes the code a bit more readable for contributors.

Reply:
@adamjmcgrath Regardless of the other comments in this PR, can you address this one?

Reply:
Yep - I'll address this one this week

Reply:
1b41448 - sorry for the delay